packages/daimo-api/src/server/appVersion.ts

```diff
@@ -0,0 +1,49 @@
+import { assertEqual } from "@daimo/common";
+
+let appVersionTracker = null as AppVersionTracker | null;
+
+export function getAppVersionTracker() {
+  if (appVersionTracker == null) {
+    appVersionTracker = new AppVersionTracker();
+  }
+  return appVersionTracker;
+}
+
+export class AppVersionTracker {
+  private latestVersion = null as string | null;
+
+  // Start polling for app version updates
+  init() {
+    // Check for updates every 5 minutes
+    this.checkVersion();
+    setInterval(() => this.checkVersion(), 5 * 60 * 1000);
+  }
+
+  getLatestVersion() {
+    return this.latestVersion;
+  }
+
+  // Check for updates
+  async checkVersion() {
+    try {
+      this.latestVersion = await this.getLatestVersionIOS();
+      console.log(
+        `[APP-VERSION] got latest iOS version: ${this.latestVersion}`
+      );
+    } catch (e) {
+      console.error(`[APP-VERSION] error checking iOS version`, e);
+    }
+  }
+
+  async getLatestVersionIOS() {
+    const url = "https://itunes.apple.com/lookup?bundleId=com.daimo";
```

Comment:
maybe we should read the android version separately?
Once we submit app to review it could take variable time to land on Android vs iOS and it might prompt banner for users without an actual update.

Reply:
will do in a followup.. getting the latest android version is harder.